docs/onboarding-extra-resources.md

```diff
@@ -108,6 +108,11 @@ You will be required to add:
   explained early
 * _Description_: a short description of the extra resource. Please try to be
   clear and brief
+* _Payment option_: when - and how much for - will the extra resource be billed:
+  * __Pre-paid__: in full at the start of the billing period
+  * __Post-paid__: at the end of the billing period, according to its effective
```

Comment:
in full at the end of the billing period, or proportional to the effective usage if terminated early